repose-aggregator/components/filters/openstack-identity-v3-filter/src/main/scala/org/openrepose/filters/openstackidentityv3/utilities/OpenStackIdentityV3API.scala

```diff
@@ -199,7 +201,10 @@ class OpenStackIdentityV3API(config: OpenstackIdentityV3Config, datastore: Datas
                 // TODO: Come up with a better algorithm to decide the cache TTL and handle negative/0 TTLs
                 val ttl = if (offsetConfiguredTtl < 1) identityTtl else math.max(math.min(offsetConfiguredTtl, identityTtl), 1)
```

Comment:
Pointing out where the ttl is determined, in case @adrianjgeorge wants to take a look at the code itself.